state/upgrade.go

```diff
@@ -330,7 +330,7 @@ func (st *State) EnsureUpgradeInfo(
 			Assert: txn.DocExists,
 		})
 	}
-	if err := st.runRawTransaction(ops); err == nil {
+	if err := st.db().NoChangeLog().RunRawTransaction(ops); err == nil {
```

Comment:
Yeah so as per the interface comment, can we wrap a st.db() here to add the nochangelog behaviour